### ISSUE TITLE
Bug: Home page does not load if a user has no permissions to data

### ISSUE THREAD

**Issue by vitorhirota** [metabase/metabase#10212]
## Steps to reproduce:
- For a given group, revoke access to all databases 
- Access metabase home page, eg, http://localhost:3000
- Page will show the header, and the loading gif indefinitely

## Workaround:
Give access to a single table, home page works as expected.

## Aditional notes
Just upgrade from 0.31.x, permissions to only collections and no databases worked fine.

## System configuration / info
- Your browser and the version: Chrome 74
- Your operating system: Windows 10
- Your databases: Postgres, Oracle
- Metabase version: 0.32.9
- Metabase hosting environment: Ubuntu
- Metabase internal database: Postgres

**Comment by flamber** [comment]
This is generating the same error as #10191 - and a couple of extra ones.
```
Error: Minified React error #143
```
```
TypeError: e.parentNode is null
```
```
Unhandled promise rejection TypeError: "n is null"
```
```
Unhandled promise rejection TypeError: "a[0] is undefined"
```

**Comment by salsakran** [comment]
Hello all, this is fixed in 0.32.10 which just went out.
